apps/nextra/pages/en/build/cli/trying-things-on-chain/ledger.mdx

```diff
@@ -117,3 +117,266 @@ yes
 After you have approved publishing this package you will be prompted to sign the transaction on your Ledger device. Once signed, the package will be published to the network!
 
 One error you might run into is `Error: Wrong raw transaction length`. This means that the transaction or package size was too big for your device to sign. Currently the Aptos Ledger app can only support transactions that are smaller than 20kb. The `Ledger Nano S` device has less memory than that, which is why it is more likely to produce this error.
+
+## Authentication key rotation
+
+If you have an active account that is not secured using a hardware wallet, then you may wish to rotate the account's authentication key so that it is linked with your Ledger.
+
+Alternatively, if you have an account linked with a Ledger hardware wallet that you wish to publish a large package from, you might want to temporarily rotate the account's authentication key to a hot key.
+
+This tutorial will walk you through both scenarios.
+
+
+<Callout type="default" emoji="🧠">
+Before you start this tutorial make sure you have `jq`, a lightweight and flexible command-line JSON processor:
+
+```sh filename="Terminal"
+jq --version
+```
+</Callout>
+
+<Steps>
+
+### Start a localnet
+
+You can run this command at any point to restart a fresh localnet:
+
+```sh filename="Terminal"
+mkdir -p localnet-data
+aptos node run-localnet \
+    --assume-yes \
+    --test-dir localnet-data \
+    --force-restart
+```
+
+The localnet is ready when it prints out:
+
+```sh filename="Terminal"
+Applying post startup steps...
+
+Setup is complete, you can now use the localnet!
+```
+
+### Move localnet to a background process
+
+Open up a new terminal window to continue with the rest of the tutorial and leave the localnet running in the background.
+
+<Callout type="default">
+When you finally want to shut down the localnet, you can press `Ctrl+C` in the terminal window where you started the localnet.
+</Callout>
+
+### Set up localnet hot wallet profile
+
+In a fresh terminal window, set up a localnet profile for an account with a compromised private key:
```

Comment:
I wouldn't make this a compromised key, I was just create a new profile with a random key for the purposes of the tutorial.  Either that or just say with a "fixed private key" for the purposes of the tutorial.

Compromised shouldn't really come into play here.  The only reason it is compromised is because it's hardcoded a paragraph down.

Reply:
Removed as of 6b6cb8c